homeassistant/components/velbus/quality_scale.yaml

```diff
@@ -0,0 +1,70 @@
+rules:
+  # Bronze
+  action-setup: todo
+  appropriate-polling:
+    status: exempt
+    comment: |
+      This integration does not do polling.
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
+  config-flow: todo
+  dependency-transparency: done
+  docs-actions: todo
+  docs-high-level-description: todo
+  docs-installation-instructions: todo
+  docs-removal-instructions: todo
+  entity-event-setup: todo
+  entity-unique-id: done
+  has-entity-name: todo
+  runtime-data: todo
+  test-before-configure: done
+  test-before-setup: todo
+  unique-config-entry: done
```

Comment:
Why does usb get unique id and manual setup only abort on common port?

Reply:
this seems to be some legacy implementation, i already have a fix locally, will push it aftert this PR